source/index.html.md

```diff
@@ -128,7 +130,7 @@ message being sent. Let's start with one of the simplest triggers: WorldLoad. In
 order to register a trigger, we use the provided `register` function. It takes
 the trigger name as the first argument (case-insensitive), and the function to
 run as the second argument. You can see a complete list of these triggers on our
-[javadocs, under `IRegister`](https://www.chattriggers.com/javadocs/com/chattriggers/ctjs/engine/IRegister.html).
+[javadocs](https://www.chattriggers.com/javadocs/), under [`IRegister`](https://www.chattriggers.com/javadocs/com/chattriggers/ctjs/engine/IRegister.html).
```

Comment:
That link is dead, it should be `https://chattriggers.com/javadocs/-chat-triggers/com.chattriggers.ctjs.engine/-i-register/index.html`